content/src/content/jcr_root/apps/core/wcm/components/commons/datalayer/clientlibs/js.txt

```diff
@@ -0,0 +1,21 @@
+###############################################################################
+# Copyright 2019 Adobe
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+#base=js
+
+datalayer.js
+consumer.js
```

Comment:
we should really put the mock files before the datalayer. Might even be better to have two clientlibs that aren't embedded:
• `core.wcm.components.commons.datalayer.mocks`
• `core.wcm.components.commons.datalayer`
and include them in that order